manuscript/250_Value_Objects.md

```diff
@@ -24,7 +24,7 @@ Let's say that we have indeed decided to use a `decimal` to hold a price, and a
 
 ### Time passes...
 
-Actually, it turns out that these values must be shared across few subdomains of the system. For example:
+It turns out that these values must be shared across a few subdomains of the system. For example:
```

Comment:
I now think it would be better to say "One day, it turns out...". What do you think?